pkg/resolver/search.go

```diff
@@ -107,6 +123,86 @@ func (s *SearchResult) Uids() {
 	s.resolveUids()
 }
 
+// Get a copy of the current user access if user data exists
+func userdataExists(ctx context.Context) (*UserResourceAccess, error) {
```

Comment:
I'd rename to `getUserAccessData()`.  Return an error if the data doesn't exist.

Reply:
function removed.